ParsedChatCompletion not found in 'openai/resources/beta/chat/completions.mjs'

### Example Code

I get the following error ```node_modules/@langchain/openai/dist/chat_models.d.ts:9:38 - error TS7016: Could not find a declaration file for module 'openai/resources/beta/chat/completions.mjs'.```
for ```import { ParsedChatCompletion } from "openai/resources/beta/chat/completions.mjs";```


### Error Message and Stack Trace (if applicable)

_No response_

### Description

I'm trying to build my nodejs project that includes the langchain package

### System Info

Using:
node v20.13.1
npm 10.5.2
"langchain": "^0.2.17"
"openai": "4.57.0"

--- a/libs/langchain-openai/package.json
+++ b/libs/langchain-openai/package.json
@@ -37,7 +37,7 @@
   "dependencies": {
     "@langchain/core": ">=0.2.26 <0.3.0",
     "js-tiktoken": "^1.0.12",
-    "openai": "^4.55.0",
+    "openai": "^4.57.3",
     "zod": "^3.22.4",
     "zod-to-json-schema": "^3.22.3"
   },

--- a/libs/langchain-openai/src/chat_models.ts
+++ b/libs/langchain-openai/src/chat_models.ts
@@ -61,7 +61,6 @@ import type {
   ResponseFormatJSONObject,
   ResponseFormatJSONSchema,
 } from "openai/resources/shared";
-import { ParsedChatCompletion } from "openai/resources/beta/chat/completions.mjs";
 import type {
   AzureOpenAIInput,
   OpenAICallOptions,
@@ -1615,7 +1614,8 @@ export class ChatOpenAI<
   async betaParsedCompletionWithRetry(
     request: OpenAIClient.Chat.ChatCompletionCreateParamsNonStreaming,
     options?: OpenAICoreRequestOptions
-  ): Promise<ParsedChatCompletion<null>> {
+    // Avoid relying importing a beta type with no official entrypoint
+  ): Promise<ReturnType<OpenAIClient["beta"]["chat"]["completions"]["parse"]>> {
     const requestOptions = this._getClientOptions(options);
     return this.caller.call(async () => {
       try {

